wasm.md

```diff
@@ -1031,18 +1031,18 @@ Sort `Signedness` is defined in module `BYTES`.
 
     syntax LoadOpM ::= LoadOp | LoadOp MemArg
  // -----------------------------------------
-    rule <k> ITYPE . LOP:LoadOp               => ITYPE . LOP  IDX                          ... </k>
+    rule <instrs> ITYPE . LOP:LoadOp               => ITYPE . LOP  IDX                          ... </instrs>
          <valstack> < i32 > IDX : VALSTACK    => VALSTACK </valstack>
-    rule <k> ITYPE . LOP:LoadOp MEMARG:MemArg => ITYPE . LOP (IDX +Int #getOffset(MEMARG)) ... </k>
+    rule <instrs> ITYPE . LOP:LoadOp MEMARG:MemArg => ITYPE . LOP (IDX +Int #getOffset(MEMARG)) ... </instrs>
          <valstack> < i32 > IDX : VALSTACK    => VALSTACK </valstack>
 
-    rule <k> load { ITYPE WIDTH EA SIGN }
+    rule <instrs> load { ITYPE WIDTH EA SIGN }
           => < ITYPE > #if SIGN ==K Signed
                            #then #signedWidth(WIDTH, #getRange(DATA, EA, WIDTH))
                            #else #getRange(DATA, EA, WIDTH)
                        #fi
          ...
```

Comment:
Formatting, line up `=>` and `...` and `<instrs>` on the right side.

Reply:
Done